### PR TITLE
Update dashboard to work without SKU

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it:
- updates dashboard to work without SKU
- allow to export variant ID in products export

Backend: https://github.com/mirumee/saleor/pull/7633

**PR intended to be tested with API branch:**  feature/empty-sku

### Screenshots

Zrzut ekranu 2021-09-27 o 20 57 10
Zrzut ekranu 2021-09-27 o 15 30 25

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://feature-empty-sku.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4499 Update frontend to work without SKU](https://app.clickup.com/t/2549495/SALEOR-4499)

**Review by rafalp — APPROVED** [review]

File: src/orders/components/OrderProductsCardElements/OrderProductsTableRow.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ const TableLine: React.FC<TableLineProps> = ({
         {maybe(() => line.orderLine.productName) || <Skeleton />}
       </TableCellAvatar>
       <TableCell className={classes.colSku}>
-        {line?.orderLine.productSku || <Skeleton />}
+        {line?.orderLine ? line.orderLine.productSku : <Skeleton />}
```
Comment (bmigirl) [inline src/orders/components/OrderProductsCardElements/OrderProductsTableRow.tsx:99]: As I undestand `line.orderLine.productSku` can still be undefined, even though the `line?.orderLine` exists. What should be displayed then?

Comment (orzechdev) [reply src/orders/components/OrderProductsCardElements/OrderProductsTableRow.tsx:99]: Then nothing should be displayed, empty table cell.

**Review by kamilpastuszka — CHANGES_REQUESTED** [review]
I'm getting unspecified error when I attempt to  export variant ID.

unknown error when exporting variant ID

**Comment by orzechdev** [comment]
> I'm getting unspecified error when I attempt to export variant ID.
> 

Please check with `feature/empty-sku` branch

**Review by bmigirl — APPROVED** [review]

**Review by kamilpastuszka — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**